### PR TITLE
Only do profiling on unix systems

### PR DESCRIPTION
**Pull request by mads256h**
Use a dummy implementation of `FlamegraphProfiler` on not-unix platforms.

### CODE REVIEW

**Review by t-lohse — DISMISSED** [review]

**Review by henneboy — APPROVED** [review]
Looks good to me, had my eyes closed though

File: benches/flamegraph/flamegraph_profiler.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    }
+}
+
+impl Profiler for FlamegraphProfiler {
+    fn start_profiling(&mut self, _benchmark_id: &str, _benchmark_dir: &Path) {
+        // Not implemented
+    }
+
+    fn stop_profiling(&mut self, _benchmark_id: &str, _benchmark_dir: &Path) {
+        // Not implemented
```
Comment (henneboy) [inline benches/flamegraph/flamegraph_profiler.rs:19]: use unimplemented macro

File: benches/flamegraph/flamegraph_profiler.rs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+impl FlamegraphProfiler {
+    pub fn new(_frequency: c_int) -> Self {
+        FlamegraphProfiler {}
+    }
+}
+
+impl Profiler for FlamegraphProfiler {
+    fn start_profiling(&mut self, _benchmark_id: &str, _benchmark_dir: &Path) {
+        // Not implemented
```
Comment (henneboy) [inline benches/flamegraph/flamegraph_profiler.rs:15]: use unimplemented macro

**Review by henneboy — CHANGES_REQUESTED** [review]
Opened my eyes

File: benches/flamegraph/flamegraph_profiler.rs (hunk 1 above)
Comment (mads256h) [reply benches/flamegraph/flamegraph_profiler.rs]: fixed

File: benches/flamegraph/flamegraph_profiler.rs (hunk 2 above)
Comment (mads256h) [reply benches/flamegraph/flamegraph_profiler.rs]: fixed

**Review by henneboy — APPROVED** [review]

**Review by HTHNLNDN — COMMENTED** [review]
Very nice

**Pull request merged**